galore/cross_sections.py

```diff
@@ -281,3 +281,147 @@ def _eval_fit(energy, coeffs):
                 orb: _eval_fit(energy, np_fromstr(coeffs))
                 for orb, coeffs in orbitals_fits}})
     return el_cross_sections
+
+
+
+import tarfile
+import numpy as np
+def read_csv_file(tar_file_name,file_path):
+    '''read csv file 
+    Input:  the file name
```

Comment:
Please use "Args:" and "Returns:", indicating the type e.g.
```
Args:
    tar_file_name (str): path to tarfile of CSV data
    file_path (str): path to individual CSV file within tarfile
  
  Returns:
    dict containing 'headers', 'electron_counts' (lists of str and int respectively) and 'data_table', a 2-D nested list of floats. Missing data is represented as None.```

Ideally people would be able to read the docstring and then understand how to use the function without reading the source code. For small, private functions this may be unnecessary/overkill, but for something more complicated like a parser it is very difficult to understand what promises are being made about the input/output format simply by glancing at the code.